Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -222,8 +238,15 @@ const std::vector<std::unique_ptr<VideoBackendBase>>& VideoBackendBase::GetAvail
     backends.push_back(std::make_unique<DX12::VideoBackend>());
 #endif
 #ifdef HAS_VULKAN
+#ifdef __APPLE__
+    if (!is_vulkan_preferred)
+    {
+      backends.push_back(std::make_unique<Vulkan::VideoBackend>());
+    }
```

Comment:
```suggestion
#ifdef __APPLE_
    if (__builtin_available(macOS 10.14, *))
    {
      backends.emplace(backends.begin(), std::make_unique<Vulkan::VideoBackend>());
    }
    else
#endif
    {
      backends.push_bach(std::make_unique<Vulkan::VideoBackend>());
    }
```

And get rid of all the other changes except the comment block? It makes the change much more self-contained, but it does hide intent a bit. Not sure what is best way.

Reply:
I think this is definitely cleaner than how I did it and it doesn't seem to hide intent *that* much. Perhaps an extra comment will help with that.